lib/config/pricing/src/index.js

```diff
@@ -69,13 +69,6 @@ const error = (reason) => ({
 const plan = (ppid, country, auth, cb) => {
   debug('Retrieving pricing plan %s', ppid);
 
-  if(!ppid || !country) {
-    debug('Undefined pricing plan id / country');
-    let eMessage = !ppid ? 'Undefined pricing plan id' : '';
-    eMessage += !country ? ' Undefined pricing country' : '';
```

Comment:
I'd suggest to have two different error messages here (and two different returns) instead of trying to combine the two (which could be another possible approach but then we'd need to generalize it to all the other cases as well).